be/src/http/action/transaction_stream_load.cpp

```diff
@@ -104,7 +104,7 @@ void TransactionManagerAction::handle(HttpRequest* req) {
 
     if (boost::iequals(txn_op, TXN_BEGIN)) {
         st = _exec_env->transaction_mgr()->begin_transaction(req, &resp);
-    } else if (boost::iequals(txn_op, TXN_COMMIT)) {
+    } else if (boost::iequals(txn_op, TXN_COMMIT) || boost::iequals(txn_op, TXN_PREPARE)) {
```

Comment:
what's the difference between prepare failed and commit failed?

Reply:
the result is same, the transaction will be fail and rollback.